docs/usage.rst

```diff
@@ -145,6 +149,60 @@ subset of the files within the that directory with the optional ``config['patch_
 All assets defined in ``config['patch_files']`` will overwrite any files specified in the
 initial conditions or forcing if they have the same target location and name.
 
+DiagTable configuration
+-----------------------
+
+The ``diag_table`` specifies the diagnostics to be output by the Fortran model. See documentation
+for the string representation of the ``diag_table``
+`here <https://mom6.readthedocs.io/en/latest/api/generated/pages/Diagnostics.html>`_. The fv3config
+package defines a python representation of this object, :py:class:`~fv3config.DiagTable`, which can
+be used to explicitly represent the ``diag_table`` within an fv3config configuration dictionary.
+
+The ``DiagTable`` object can be initialized from a dict as follows:
+
+.. code-block:: python
+
+    >>> import fv3config
+    >>> from datetime import datetime
+    >>> diag_table_dict = {
+        "name": "example_diag_table",
+        "base_time": datetime(2000, 1, 1),
+        "files": [
+            {
+                "name": "physics_diagnostics",
+                "frequency": 1,
+                "frequency_units": "hours",
+                "fields": [
+                    {
+                        "module_name": "gfs_phys",
+                        "field_name": "totprcpb_ave",
+                        "output_name": "surface_precipitation_rate",
+                    },
+                    {
+                        "module_name": "gfs_phys",
+                        "field_name": "ULWRFtoa",
+                        "output_name": "upward_longwave_radiative_flux_at_toa",
+                    }
+                ]
+            }
+        ]
+    }
+    >>> diag_table = fv3config.DiagTable.from_dict(diag_table_dict)
+    >>> print(diag_table)  # will output diag_table format expected by Fortran model
+    example_diag_table
+    2000 1 1 0 0 0
+
+    "physics_diagnostics", 1, "hours", 1, "hours", "time"
+
+    "gfs_phys", "totprcpb_ave", "surface_precipitation_rate", "physics_diagnostics", "all", "none", "none", 2
+    "gfs_phys", "ULWRFtoa", "upward_longwave_radiative_flux_at_toa", "physics_diagnostics", "all", "none", "none", 2
+
+String representations of the `diag_table` can be parsed with the :py:meth:`fv3config.DiagTable.from_str` method.
+
+If serializing an `fv3config` configuration object to yaml, it is recommended to use the
+:py:meth:`fv3config.DiagTable.asdict` method before dumping the configuration dictionary. This is implemented
+by :py:meth:`fv3config.config_to_yaml`.
```

Comment:
When you say `implemented by`, do you mean `called by`?

Can this be shown as an example? It would be pretty short, and makes it clear whether it's e.g. `yaml.dump(diag_table.asdict())` or `yaml.dump(DiagTable.asdict(diag_table))`.